upstash/redis/database/crud.go

```diff
@@ -0,0 +1,79 @@
+package database
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/upstash/terraform-provider-upstash/upstash/client"
+	"github.com/upstash/terraform-provider-upstash/upstash/utils"
+)
+
+func resourceDatabaseUpdate(ctx context.Context, data *schema.ResourceData, m interface{}) diag.Diagnostics {
+	c := m.(*client.UpstashClient)
+	databaseId := data.Get("database_id").(string)
+	if data.HasChange("multizone") {
```

Comment:
How does it work? If can user disable multizone? What happens if they do?

Reply:
Similar to ```tls```, see the comment above.